collect_app/src/main/java/org/odk/collect/android/formmanagement/FormDeleter.java

```diff
@@ -33,7 +29,5 @@ public void delete(Long id) {
         } else {
             formsRepository.softDelete(form.getDbId());
         }
-
-        fastExternalItemsetsRepository.deleteAllByCsvPath(form.getFormMediaPath() + File.separator + "itemsets.csv");
```

Comment:
We could create an issue, but when @lognaturel and I talked it about it, it seemed like it would be a "Won't fix" anyway. @lognaturel is this something we'd prioritize in the next couple of years?

Reply:
I’d be surprised if it’s something we three prioritize but it’s not a bad idea to have an issue so that if a user runs into problems it’s easy to find and so that a contributor can possibly take it on.

Reply:
Ok issue [here](https://github.com/getodk/collect/issues/4599).